metrics/metricskey/constants.go

```diff
@@ -29,18 +29,18 @@ const (
 	// LabelNamespaceName is the label for immutable name of the namespace that the service is deployed
 	LabelNamespaceName = "namespace_name"
 
-	// ContainerName is the container for which the metric is reported.
-	ContainerName = "container_name"
```

Comment:
Why are these two vars removed? They are used in downstream repos, for example: https://github.com/knative/serving/blob/master/pkg/metrics/key.go#L35